<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCraftMeasure</class>
 <widget class="QDialog" name="DialogCraftMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>404</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>请选择测试信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
     <item row="0" column="0">
      <widget class="QLabel" name="labelMeasureType">
       <property name="text">
        <string>测试类型：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxMeasureType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelMeasureName">
       <property name="text">
        <string>测试名称：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditMeasureName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelUnit">
       <property name="text">
        <string>测试单位：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditUnit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelMeasureSerial">
       <property name="text">
        <string>测试序号：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditMeasureSerial"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelStandardValue">
       <property name="text">
        <string>标准值：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditStandardValue"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelUpperLimit">
       <property name="text">
        <string>上限值：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditUpperLimit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelLowerLimit">
       <property name="text">
        <string>下限值：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditLowerLimit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAccept">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReject">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
